qml/keys/ExtendedListSelector.qml

```diff
@@ -33,7 +33,7 @@ Item {
     property int currentlyAssignedKeyX: currentlyAssignedKey ? currentlyAssignedKey.x : 0
     property int currentlyAssignedKeyY: currentlyAssignedKey ? currentlyAssignedKey.y : 0
 
-    property bool isTwoLines: keyRepeater.count>5
+    property int numberOfLines: keyRepeater.count <= 4 ? 1 : Math.floor(keyRepeater.count / 4) + 1
```

Comment:
no need for a ternary operator: "Math.floor(keyRepeater.count / 4) + 1" is always valid.